charts/k8ssandra/templates/stargate/ingress.yaml

```diff
@@ -78,39 +40,25 @@ spec:
               servicePort: 8080
   {{- if $traefik.stargate.graphql.playground.enabled }}
           - path: /playground
+            pathType: Exact
```

Comment:
Out of curiosity why did you use `pathType: Exact` here?

Reply:
My goal here was to minimize the footprint that this route makes in the overall ingress landscape, to leave as much room as possible for other software to coexist with k8ssandra. It's a hedge against the fact that native ingress routes are not port-sensitive.

It's a small thing, and if you wanted to argue that I should remove it, I'd not put up a fight.

Reply:
IIRC it's a single page app, there should be no routing beyond `/playground`. With `pathType: Exact` a request to `/playground/foo` will _not_ be routed to Stargate.

With that being said I have not personally tested the routes Stargate uses for the playground section, thus I don't know if its usage is appropriate here.